release notes/v0.49.0.md

```diff
@@ -17,9 +17,13 @@ k6 `v0.49.0` is here 🎉! This release includes:
 
 _optional intro here_
 
-### `<big_feature_1>` `#pr`
+### Add tracing to browser module [browser#1100](https://github.com/grafana/xk6-browser/pull/1100)
 
-_what, why, and what this means for the user_
+The browser module now generates traces that provide a representation of its inner workings, such as API methods executed (for example `browser.newPage` and `page.goto`), page navigations, and [Web Vitals](https://grafana.com/docs/k6/latest/using-k6-browser/metrics/#googles-core-web-vitals) measurements.
+
+Take into account that currently the instrumented methods are a subset of all the methods exposed by the browser module API, but this will be extended in the future.
```

Comment:
It felt like the word `currently` was enough to help highlight the point. Happy either way though.

```suggestion
Currently the instrumented methods are a subset of all the methods exposed by the browser module API, but this will be extended in the future.
```